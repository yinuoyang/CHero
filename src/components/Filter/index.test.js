import React from 'react';
import { Filter } from './index';
import renderer from 'react-test-renderer';
import testData from '../../static/testData/fakeData.json'


describe ('Test Filter component with data', () => {
    let component;
    beforeEach(() => {
         component =renderer.create(
            <Filter suppliers = {testData.data}  filterSelectedSupplier={() => {}} resetFilter={()=> {}} />
        ) 
    })

    it( 'Component render the component match the snapshot', ()=> {
        expect(component.toJSON()).toMatchSnapshot();
    })

    

})
