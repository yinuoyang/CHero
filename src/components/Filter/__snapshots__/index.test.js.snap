// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Filter component with data Component render the component match the snapshot 1`] = `
<div
  className="FilterSection"
>
  <div
    className="FilterMenu"
  >
    <div
      className="inputBox"
    >
      <input
        className="FilterInput"
        onClick={[Function]}
        readOnly={true}
        value="All Suppliers"
      />
    </div>
    <button
      className="resetButton"
      onClick={[Function]}
    >
      Reset Filter
    </button>
  </div>
</div>
`;
